content/md/first-steps/manage-data-quality.md

```diff
@@ -0,0 +1,30 @@
+---
+id: manage-data-quality
+themes: catalog-settings
+title: Manage your **data quality**
+popular: false
+ee-only: true
+related: understand-data-quality, improve-data-quality
+---
+
+# Overview
+
+In order to check, score and get recommendations on the product data that truly matters to craft a great product experience,  you can disable Data Quality Insights calculation for non relevant attribute groups (ie. ERP, internal…)
+By doing so, you will be able to highlight the quality of the attributes that are the most valuable for your customers.
+
+# Data quality insights activation and deactivation
```

Comment:
In this title, you mention "activation", but below you only explain how to deactivate an attribute group. It sounds weird. I think you should either create two sections ("Activation" and "Deactivation") or, explain how to activate in this section as well. Separating into two sections being my favorite option. ;)